src/client/app/components/groups/CreateGroupComponent.tsx

```diff
@@ -107,8 +166,52 @@ class CreateGroupComponent extends React.Component<CreateGroupPropsWithIntl, {}>
 		}
 	}
 
+	private handleGPSChange(e: React.ChangeEvent<HTMLInputElement>) {
+		this.setState({ gpsInput: e.target.value });
+	}
+
+	private handleDisplayChange(e: React.ChangeEvent<HTMLInputElement>) {
+		this.props.editGroupDisplayable(e.target.value === 'true');
+	}
+
+	private handleNoteChange(e: React.ChangeEvent<HTMLTextAreaElement>) {
+		this.props.editGroupNote(e.target.value);
+	}
+
+	private handleAreaChange(e: React.ChangeEvent<HTMLInputElement>) {
+		this.setState({ groupArea: e.target.value });
+		// still need to set state to parse check before submitting
+	}
+
 	private handleCreateGroup() {
-		this.props.submitGroupInEditingIfNeeded();
+		const gpsProxy = this.state.gpsInput.replace('(','').replace(')','').replace(' ', '');
+		const pattern2 = /^\d+(\.\d+)?$/;
+		// need to check gps and area
+		// gps and area are still optional so check if blank
+		if (this.state.groupArea.match(pattern2) || this.state.groupArea === '') {
+			if (this.state.groupArea !== '') {
+				this.props.editGroupArea(parseFloat(this.state.groupArea));
+			}
+			if (this.state.gpsInput === '' || isValidGPSInput(gpsProxy)) {
+				if (this.state.gpsInput !== '') {
+					// if it satisfies if condition, and defined, then set GPSPoint
+					const parseGPS = gpsProxy.split(',');
+					// should only have 1 comma
+					const gPoint: GPSPoint = {
+						longitude: parseFloat(parseGPS[0]),
+						latitude: parseFloat(parseGPS[1])
+					};
+					this.props.editGroupGPS(gPoint);
+				}
+				this.props.submitGroupInEditingIfNeeded();
+			}
+			else {
+				window.alert(this.props.intl.formatMessage({id: 'group.gps.error'}));
```

Comment:
isValidGPSInput creates a popup if there is an issue witht he GPS format. This line also does that so you get a second popup. I tried to think of a case where removing this popup would mean no message but I could not quickly get one. If this is correct, then I think it best to remove this one.

Reply:
removed else statement for both